sc-network/sctp_client/CMakeLists.txt

```diff
@@ -21,11 +21,16 @@ else ()
         "sockets/glibSocket.cpp"
         "sockets/glibSocket.hpp"
     )
-    set (SOCKET_LIBS  ${GLIB2_LIBRARIES})
+    set (SOCKET_LIBS  ${GLIB2_INCLUDE_DIRS})
 endif()
 
 add_library (sctp-client SHARED ${SOURCES} ${HEADERS} ${SOCKET_SRC})
-target_link_libraries(sctp-client sc-memory-cpp ${SOCKET_LIBS})
+
+target_include_directories(sctp-client
+  PRIVATE ${SC_MEMORY_SRC}
+  PRIVATE ${SOCKET_LIBS}
+  )
+target_link_libraries(sctp-client sc-memory ${SOCKET_LIBS})
```

Comment:
```suggestion
target_link_libraries(sctp-client sc-memory ${GLIB2_LIBRARIES})
```

Reply:
Left `SOCKET_LIBS` here as for win32 glib2 will not be set

Reply:
But it is empty for windows
```
if (${WIN32})
    set (SOCKET_SRC
        "sockets/winSocket.cpp"
        "sockets/winSocket.hpp"
    )
    set (SOCKET_LIBS
    )
else ()
```

Reply:
Yes, it was initially empty. I expect that because of glib not needed for win32 and to not set glib it was empty. I can't check sc-network build for win32, so just left these lines as it was. My change here is an addition of `target_include_directories`